src/main/java/org/wirvsvirus/locoronando/dealer/Dealer.java

```diff
@@ -0,0 +1,22 @@
+package org.wirvsvirus.locoronando.dealer;
+
+import lombok.Data;
+
+import javax.persistence.Embeddable;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@Entity
+@Data
+public class Dealer {
+  @Id
+  @GeneratedValue
+  private long id;
+  private String name;
+  private String category;
```

Comment:
And should we have category objects? Otherwise searching for them will be difficult...